Response/DatatableQueryBuilder.php

```diff
@@ -261,7 +269,8 @@ private function initColumnArrays()
                     $previousAlias = $currentAlias;
 
                     $currentPart = array_shift($parts);
```

Comment:
You should do `$this->getSafeName(array_shift($parts));` here

Reply:
This change result in the following error

![image](https://user-images.githubusercontent.com/26867068/61205640-f12b0100-a6f0-11e9-8042-4fcc8bb1c9ad.png)

Reply:
This should not be required since you added `$currentAlias = $this->getSafeName($currentAlias);` later.
Is the actual code fix your issue?

Reply:
@Seb33300 The actual code fixes the issue indeed, when we apply the suggested solution the error posted by @darylholling occurs.